### PR TITLE
chore: prepare for 1.5.11 release

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?

Bump executor to 0.5.11

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Review by byawitz — APPROVED** [review]

File: app/views/install/compose.phtml [hunk 1]
```diff
@@ -753,7 +753,9 @@ $image = $this->getParam('image', '');
     <<: *x-logging
     restart: unless-stopped
     stop_signal: SIGINT
-    image: openruntimes/executor:0.5.7
+    image: openruntimes/executor:0.5.9
+    labels:
+      openruntimes-image: 0.5.9
```
Comment (stnguyen90) [inline app/views/install/compose.phtml:758]: Why do we need this?

File: app/views/install/compose.phtml [hunk 2]
```diff
@@ -771,6 +773,7 @@ $image = $this->getParam('image', '');
       - OPR_EXECUTOR_DOCKER_HUB_USERNAME=$_APP_DOCKER_HUB_USERNAME
       - OPR_EXECUTOR_DOCKER_HUB_PASSWORD=$_APP_DOCKER_HUB_PASSWORD
       - OPR_EXECUTOR_ENV=$_APP_ENV
+      - OPR_EXECUTOR_IMAGE=0.5.9
```
Comment (stnguyen90) [inline app/views/install/compose.phtml:776]: we need this too?

**Review by stnguyen90 — APPROVED** [review]
Left some comments, but otherwise, it works well. I was able to create an execution:

image

and build logs were as expected:

image

File: app/views/install/compose.phtml (hunk 1 above)
Comment (christyjacob4) [reply app/views/install/compose.phtml]: removed 👍

File: app/views/install/compose.phtml (hunk 2 above)
Comment (christyjacob4) [reply app/views/install/compose.phtml]: removed 👍

**Pull request merged**